docs/src/development.md

```diff
@@ -471,14 +471,16 @@ python release.py --help
 
 ### Decide the release type
 
-*Quality-time* adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html), so first you need to decide on the type of release you want to create:
+*Quality-time* adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html), so first you need to decide on the type of release you want to create, conform the [release policy](versioning.md#major-minor-and-patch-releases).
 
-- Create a **major** release if the next release contains backwards incompatible changes, and optionally other changes and bug fixes.
-- Create a **minor** release if the next release contains non-breaking changes, and optionally bug fixes.
-- Create a **patch** release if the next release contains bug fixes only.
+- Create a **major** release if the docker-composition needs to be changed to deploy the next release
+- Create a **minor** release if the next release contains new or changed functionality.
+- Create a **patch** release if the next release contains only bug fixes.
 
 If you want to test the release (for example, deploy it to a test environment, or roll out a release to early adopters), it's possible to create a **release candidate** for a major, minor, or patch release.
 
+If the release type is major, minor, or patch, update the [version overview](versioning.md#version-overview).
```

Comment:
This seems superfluous, shouldn't we only update in case of major and minor?

Reply:
No, because each (non-rc) version is listed in the version overview.